src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const employeesList = document.querySelector('ul');
+const employees = [...document.querySelectorAll('li')];
+
+function salaryNum(salary) {
```

Comment:
functions should be named with verbs - convertSalaryToNumber, parseSalaryValue, etc...